cmd/zz_gen_cmd_dnshelp.go

```diff
@@ -142,6 +143,22 @@ func displayDNSHelp(name string) error {
 		ew.writeln()
 		ew.writeln(`More information: https://go-acme.github.io/lego/dns/auroradns`)
 
+	case "autodns":
+		// generated from: providers/dns/autodns/autodns.toml
+		ew.writeln(`Configuration for Autodns.`)
+		ew.writeln(`Code:	'autodns'`)
+		ew.writeln(`Since:	'v3.1.0'`)
+		ew.writeln()
+
+		ew.writeln(`Credentials:`)
+		ew.writeln(`	- "AUTODNS_API_PASSWORD":	User Password`)
+		ew.writeln(`	- "AUTODNS_API_USER":	User ID`)
+		ew.writeln(`	- "AUTODNS_ENDPOINT":	API endpoint URL`)
+		ew.writeln()
+
+		ew.writeln()
+		ew.writeln(`More information: https://go-acme.github.io/lego/dns/autodns`)
```

Comment:
does this page need to be created?

Reply:
I think so, yes. It is autogenerated. Maybe one of the LEGO maintainers can verify.

Reply:
this page automatically created